### PR TITLE
Remove redundant table wrapper

### PR DESCRIPTION
**Pull request by 95kach**

### CODE REVIEW

File: packages/ui/src/components/modal/modal.js [hunk 1]
```diff
@@ -44,6 +44,8 @@ class Modal extends React.Component {
 
   isToggledFromInside = false;
 
+  setNode = (ref) => (this.node = ReactDOM.findDOMNode(ref));
```
Comment (dmitriyslaym) [inline packages/ui/src/components/modal/modal.js]: check the latest docs how to create refs
https://reactjs.org/docs/refs-and-the-dom.html

File: packages/ui/src/components/modal/modal.js [hunk 2]
```diff
@@ -61,6 +63,12 @@ class Modal extends React.Component {
     }
   };
 
+  handleClick = (e) => {
+    if (!this.node || !this.node.contains(e.target)) {
```
Comment (dmitriyslaym) [inline packages/ui/src/components/modal/modal.js]: have you checked if it works if you click on some child element of the modal?

File: packages/widget-modal/cypress/integration/modal.test.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ describe('Modal TCs', () => {
 
           return cy
             .get('body')
-            .click()
+            .click({ force: true })
```
Comment (dmitriyslaym) [inline packages/widget-modal/cypress/integration/modal.test.js]: why do you need to force it? Is it overshadowed by something?

File: packages/ui/src/components/modal/modal.js (hunk 2 above)
Comment (95kach) [reply packages/ui/src/components/modal/modal.js]: yes

File: packages/widget-modal/cypress/integration/modal.test.js (hunk 3 above)
Comment (95kach) [reply packages/widget-modal/cypress/integration/modal.test.js]: cross z-index: 11
modal z-index: 10

the cross shouldn't be overflowed but.

on the page, you won't find anything except modal, but without force click, it doesn't work.

Comment (dmitriyslaym) [reply packages/widget-modal/cypress/integration/modal.test.js]: but you can click in the real application without issues, right? Is it possible, that clicking on some part of the element does not work?

File: packages/widget-modal/cypress/integration/modal.test.js [hunk 4]
```diff
@@ -55,7 +55,7 @@ describe('Modal TCs', () => {
 
           return cy
             .get('body')
-            .click({ force: true })
+            .click(60, 60)
```
Comment (95kach) [inline packages/widget-modal/cypress/integration/modal.test.js]: why 60?
can you explain what approach you used to calculate this number?

File: packages/widget-modal/cypress/integration/modal.test.js [hunk 5]
```diff
@@ -55,7 +55,7 @@ describe('Modal TCs', () => {
 
           return cy
             .get('body')
-            .click()
+            .click(60, 60)
```
Comment (dmitriyslaym) [inline packages/widget-modal/cypress/integration/modal.test.js]: I've take a look at a page in debugger (we have fixed page size in cypress config) and took any place on the page that is outside from modal (exactly what a user should do in the test scenario).

Comment (95kach) [reply packages/widget-modal/cypress/integration/modal.test.js]: I expected to see something trivial more like `.click(0, 0)`

Comment (dmitriyslaym) [reply packages/widget-modal/cypress/integration/modal.test.js]: also can be, right
such a value just looks better in debugger. but not important

**Pull request merged**